common/schema-builder/src/main/java/io/smallrye/graphql/schema/helper/BeanValidationDirectivesHelper.java

```diff
@@ -39,7 +47,9 @@ public class BeanValidationDirectivesHelper {
         addArgument("maxLength", Scalars.getIntScalar());
         addArgument("format", Scalars.getStringScalar());
         addArgument("min", Scalars.getBigIntegerScalar());
+        addArgument("minFloat", Scalars.getBigDecimalScalar());
```

Comment:
This is somewhat ugly but I think it's better to have `min` and `minFloat` separately than only use a floating-point variant, because in most cases the constraints will be whole numbers, so it would look odd to always show integer numbers as floating-point numbers